src/test/java/org/folio/utility/RestUtility.java

```diff
@@ -0,0 +1,68 @@
+package org.folio.utility;
+
+import static org.folio.utility.VertxUtility.getClient;
+
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.MultiMap;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.ext.web.client.HttpResponse;
+import java.net.URL;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class RestUtility {
+  public static final String TENANT_ID = "test_tenant";
```

Comment:
What led you to choose putting this here rather than one of the other utility classes?

Reply:
The `RestUtility` needs the `TENANT_ID` when adding it to the **REST** requests.
When I noticed that there were different (seemingly random) classes also using hardcoded tenant strings I decided to just have a single location for this. The `RestUtility` already had this and so I exposed the constant.

The `StorageTestSuite` is focused on everything in the suite.
There are tests that are outside of the suite that directly use `TENANT_ID`.
This makes `StorageTestSuite` not a good candidate because it is specialized in regards to certain tests.
I believe that the `RestUtility` is more generalized than `StorageTestSuite`.

The `VertxUtility` (now `ModuleUtility`) is where I moved all of the vertx related code that is shared between many different tests. There is no `TENANT_ID` use in any of those methods and so it does not belong there.

Like `VertxUtility`, the `KafkaUtility` is for the Kafka related shared code and is only providing start and stop methods. The `TENANT_ID` doesn't belong there as well.

A class for providing constants rather than doing double-duty with a utility is likely to be even better.
Doing this would introduce even more changes and deviations that what is already done.
I tried to keep such changes to a minimum to the degree that I could where I was able to.

Reply:
Sure, this can go down as something to sort out later